docs/news/index.html

```diff
@@ -188,11 +225,14 @@ <h2>Contents</h2>
 </div>
 
 <div class="pkgdown">
-  <p>Site built with <a href="http://hadley.github.io/pkgdown/">pkgdown</a>.</p>
+  <p>Site built with <a href="http://pkgdown.r-lib.org/">pkgdown</a>.</p>
```

Comment:
Nice, forgot this was released recently. Did you use a newer version which caused most of the diff here?

Reply:
Yeah they changed a ton of stuff. I upgraded to the newest version before doing this